### ISSUE TITLE
Where to ask for assistance for quick questions?

### ISSUE THREAD

**Issue by ajcastro** [knuckleswtf/scribe#268]
Hi @shalvah I am currently doing the plugin for the test-driven documentation approach. Sometimes I have a quick question in my mind about the codebase. I think it would be faster for me to ask you. Would it be okay to ask you if I have some quick questions? Where can I ask or contact you? Maybe we can add the github's Discussion tab

**Comment by ajcastro** [comment]
For my question, I was wondering if there is util to get or parse the docBlocks base from class name and method. 
In my tdd proposal, all docblocks should declared in test classess instead of controllers. I plan to use the same docBlock annotations. So it is just moving the docblock annotations to test classes. 

I checked the source code and the docBlocks can be retrieved base from the route e.g. `$methodDocBlock = RouteDocBlocker::getDocBlocksFromRoute($route)['method'];`. 

But I want to get docBlocks base from class and method name like, I guess this is this the right code to do that? I found it just now.
```
$class = new ReflectionClass($className);
$method = u::getReflectedRouteMethod([$className, $methodName]);
$docBlocks = [
    'method' => new DocBlock($method->getDocComment() ?: ''),
    'class' => new DocBlock($class->getDocComment() ?: ''),
];
      
```
